Jenkinsfile

```diff
@@ -26,7 +34,15 @@ elifeLibrary {
             elifeGitMoveToBranch commit, 'master'
         }
 
-        stage 'Push image', {
+        stage 'Push release image', {
```

Comment:
seems to make sense as a separate step(?)

Reply:
Good to separate, but I think unstable should go first as if there's something botched in the process it has a chance to fail on unstable first before touching the stable one.

Reply:
moved down